xarray/core/formatting.py

```diff
@@ -527,7 +527,7 @@ def diff_dim_summary(a, b):
 
 
 def _diff_mapping_repr(a_mapping, b_mapping, compat, title, summarizer, col_width=None):
-    def is_array_like(value):
+    def is_duck_array(value):
         return (
             hasattr(value, "ndim")
             and hasattr(value, "shape")
```

Comment:
should we replace the local function with `utils.is_duck_array`?

Reply:
Seems like a good idea to me.